keps/sig-storage/3333-reconcile-default-storage-class/README.md

```diff
@@ -428,8 +417,8 @@ Recall that end users cannot usually observe component logs or access metrics.
 - [ ] API .status
     - Condition name:
     - Other field:
```

Comment:
Users can see `pvc.spec.storageClassName` changing from `nil` to something after they create a default StorageClass.

Reply:
Added.